mpm_tv

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Watch the output from mpm.
+#
+# @author lightspeed393
+# @link https://github.com/cipig/mmtools
+#
+if [[ -z $1 ]]; then
+    echo "You must specify a base coin config to edit. (BTC, KMD, BCH, LTC, or DOGE)"
+    echo ""
+    echo "./mpm_tv KMD"
+    exit 0
+fi
+
+if [[ $1 == "BTC" ]] || [[ $1 == "KMD" ]] || [[ $1 == "BCH" ]] || [[ $1 == "LTC" ]] || [[ $1 == "DOGE" ]]; then
+    tail -f /tmp/mpm.log.dex.$1
+else
+    echo "Choose from BTC, KMD, BCH, LTC or DOGE."
+fi
```

Comment:
This also feels a bit strange.  I wonder if there's a way to grab the output from mpm and write to this repo's mmtools.log logfile

Reply:
Yes I can change mpm_start to do `stdbuf -oL nohup ~/mmtools/mpm_run KMD komodo > $mmtoolslogfile &`

Reply:
I'll revisit this later, maybe can get it all working using `log` function for a consistent format.